torch/csrc/jit/codegen/cuda/lower_unroll.h

```diff
@@ -75,7 +77,6 @@ class TORCH_CUDA_CU_API UnrollPass : kir::ExprMutator {
 
   void handle(Expr* expr) final;
 
- private:
```

Comment:
Not technically necessary, but I prefer to have this before `private` variables. It just looks better to me to have a clear separator between functions and variables.

Reply:
added back